lambda_uploader/package.py

```diff
@@ -233,6 +233,11 @@ def package(self, ignore=None):
         if hasattr(self, '_pkg_venv') and self._pkg_venv:
             site_packages = 'lib/python2.7/site-packages'
             lib64_site_packages = 'lib64/python2.7/site-packages'
+            if not os.path.exists('lib/python2.7'):
+                # http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html
+                # Since Lambda supports python 3.6 it is fixed at 3.6
+                site_packages = 'lib/python3.6/site-packages'
+                lib64_site_packages = 'lib64/python3.6/site-packages'
```

Comment:
I would rather make it a bit more generic so we don't have to keep changing for future versions, python3.7 etc.